aim/web/api/__init__.py

```diff
@@ -19,6 +19,15 @@ def create_app():
     from aim.web.api.projects.views import projects_router
     from aim.web.api.tags.views import tags_router
     from aim.web.api.views import general_router
+
+    # v2 API routes
+    from aim.web.api.v2.views.runs import run_router
+    from aim.web.api.v2.views.tags import tag_router
+    from aim.web.api.v2.views.experiments import experiments_router
+
+    app.include_router(run_router, prefix='/api/v2/runs')
+    app.include_router(tag_router, prefix='/api/v2/tags')
```

Comment:
Question: Do we need API versions in URL? 🤔

Reply:
I don't think so, as previous versions won't be supported, no need for versioning

Reply:
Agree

Reply:
@gorarakelyan @mihran113 for the moment we might have both old and new APIs in the same code-base. The existing one has for example endpoint prefix `/api/v1/tags/`, so the new one should be either `/api/v2/tags/` or `/api/tags/` (and all other entities as well). I'm OK with both

Reply:
Actually the new API covers nearly all the functionality of the old one, hence there is no point in keeping v1.

Reply:
@gorarakelyan, then I'll stick to the following path structure: `/api/tags/`, `/api/runs`, etc.

Reply:
@alberttorosyan awesome